client/src/components/Projects/Feedback.jsx

```diff
@@ -21,88 +21,66 @@ function Feedback({ onclickBack }) {
 		setQuality(event.target.value);
 	};
 
-	const [feedback, setFeedback] = useState(project.Feedback);
-	const handleFeedbackChange = (events) => {
-		setFeedback(event.target.value);
-	};
-
-	const [presentation, setPresentation] = useState(project.Presentation_skills);
-	const handlePresentationChange = (events) => {
-		setPresentation(event.target.value);
-	};
-
-	return (
-		<section id="feedback" className="p-8">
-			<div className="p-10">
-				<div className="information mb-3">
-					<p>Project Name: {project.Project}</p>
-					<p>
-						Design:
-						<select
-							value={design}
-							onChange={handleDesignChange}
-							disabled={project.Input_disabled}
-						>
-							<option value="1">1</option>
-							<option value="2">2</option>
-							<option value="3">3</option>
-							<option value="4">4</option>
-							<option value="5">5</option>
-						</select>
-					</p>
+    const [feedback, setFeedback] = useState(project.Feedback);
+    const handleFeedbackChange = (events) => {
+        setFeedback(event.target.value);  
+    }   
+    
+    const [presentation, setPresentation] = useState(project.Presentation_skills);
+    const handlePresentationChange = (events) => {
+        setPresentation(event.target.value);  
+    }  
 
-					<p>
-						Code Quality:
-						<select
-							value={quality}
-							onChange={handleQualityChange}
-							disabled={project.Input_disabled}
-						>
-							<option value="1">1</option>
-							<option value="2">2</option>
-							<option value="3">3</option>
-							<option value="4">4</option>
-							<option value="5">5</option>
-						</select>
-					</p>
+    return (
+        <section id="feedback">
+            <div>
+                <div className="information">
```

Comment:
looks like you deleted the tailwind styling.  Did you end up using the CSS file for styling?